### PR TITLE
feat: rolling out help me choose feat to all category pages

### PR DESCRIPTION
**Pull request by roger-in-kiva**
Help me choose feat rolled out to all category pages.
As the same condition for loan tags, this feat rolled out to all category pages too. Approved by Lauren.
Unused file component removed.

### CODE REVIEW

File: src/pages/Lend/LoanChannelCategoryPage.vue [hunk 1]
```diff
@@ -83,16 +83,12 @@ export default {
...
-				|| this.targetedLoanChannel === 'short-term-loans'
-		) {
-			this.initializeHelpmeChoose();
-		} else {
-			this.initializeLoanTags();
-		}
+		this.initializeHelpmeChoose();
+
+		// Initialize Loan Tags Experiment
+		this.initializeLoanTags();
```
Comment (dyersituations) [inline src/pages/Lend/LoanChannelCategoryPage.vue:91]: Did we get approval to roll out tags to all categories as well? That's a new result of this.

Comment (roger-in-kiva) [reply src/pages/Lend/LoanChannelCategoryPage.vue:91]: Yeah, Lauren approved this as mentioned in the description

Comment (dyersituations) [reply src/pages/Lend/LoanChannelCategoryPage.vue:91]: Oh oops, I thought that was referring to the help me choose!

File: src/pages/Lend/LoanChannelCategoryPage.vue [hunk 2]
```diff
@@ -83,16 +83,12 @@ export default {
 
 		// Add to Basket Interstitial
 		this.initializeAddToBasketInterstitial();
+
 		// Initialize Help Me Choose Experiment
-		if (this.targetedLoanChannel === 'women'
```
Comment (dyersituations) [inline src/pages/Lend/LoanChannelCategoryPage.vue:87]: Looks like `targetedLoanChannel` can be removed from the component.

Comment (roger-in-kiva) [reply src/pages/Lend/LoanChannelCategoryPage.vue:87]: Done

**Review by dyersituations — APPROVED** [review]

**Pull request merged**